src/Controller/DefaultController.php

```diff
@@ -207,21 +226,27 @@ public function islandora_object_access($op, $object, $user = NULL, AccountInter
    * the 'content' variable, or override the display by providing a theme
    * suggestion.
    *
-   * @param AbstractObject $object
+   * @param \AbstractObject $object
    *   The object.
    *
    * @return array
    *   A renderable array.
    */
-  public function islandora_print_object(AbstractObject $object) {
-   return [
-     '#title' => $object->label,
-     '#theme' => 'islandora_object_print',
-     '#object' => $object,
-   ];
+  public function islandoraPrintObject(AbstractObject $object) {
+    return [
+      '#title' => $object->label,
+      '#theme' => 'islandora_object_print',
+      '#object' => $object,
+    ];
   }
 
-  public function islandora_object_manage_access_callback($perms, $object = NULL, AccountInterface $account) {
+  /**
+   * Object management access callback.
+   */
+  // @codingStandardsIgnoreStart
+  // XXX:params with defaults should be at the end. Risky to move atm.
+  public function islandoraObjectManageAccessCallback($perms, $object = NULL, AccountInterface $account) {
```

Comment:
`$account` doesn't appear to be used in this method... could delete?
... Or possibly, make the default of `NULL` explicit, and pass `$account` in the contained `islandora_object_access()` call?